data-lineage/src/main/java/com/wipro/ats/bdre/lineage/LineageMain.java

```diff
@@ -69,16 +71,28 @@
 	private Integer processId;
 	private Long instanceId;
 	private String defaultHiveDbName;
+	private LineageQuery lineageQuery;
 
 	//	private int nextId = 0;
 	private int subquerySeq = 0;
 	private PersistenceUnit persistenceUnit;
 
-	public static void main(String[] args) throws IOException, ParseException,
+	/*public LineageMain(String query, Integer processId, Long instanceId, String defaultHiveDbName) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)